src/Munkres.jl

```diff
@@ -26,7 +26,7 @@ julia> full(matching)
  0  0  2
 ```
 """
-munkres(costMat::AbstractMatrix{T}) where {T<:Real} = munkres!(copy(costMat))
```

Comment:
this copy  is no longer necessary, 
i still left the copying operation in case the input matrix contains missings